corehq/motech/repeaters/management/commands/generate_repeater_report.py

```diff
@@ -0,0 +1,123 @@
+from datetime import datetime
+from openpyxl import Workbook
+import csv
+from couchdbkit import ResourceNotFound
+
+from django.core.management.base import BaseCommand, CommandError
+from corehq.util.log import with_progress_bar
+from corehq.motech.repeaters.models import RepeatRecord
+from corehq.motech.repeaters.dbaccessors import iter_repeat_records_by_domain, get_repeat_record_count
+
+
+class Command(BaseCommand):
+    help = """
+    Pass Repeater ID along with domain and State(Optional) or
```

Comment:
Reading just from the help message, it's not clear what the input file should be and whether the output will be a stdout or a file somewhere

Reply:
That is so true.. Updated.